.gitignore

```diff
@@ -1,3 +1,8 @@
 node_modules
 npm-debug.log*
-.DS_store
\ No newline at end of file
+.DS_store
+app/*.js
```

Comment:
None of this is essential but you're building a bunch of temporary files. You have some of them stored already (I delete them here) and also ignore some of the new ones I'm creating.